api/test-context/src/test_context.rs

```diff
@@ -116,17 +102,21 @@ pub fn new_test_context(test_name: String, api_version: &str) -> TestContext {
     let (validator_identity, _, _) = validators[0].get_key_objects(None).unwrap();
     let validator_owner = validator_identity.account_address.unwrap();
 
-    let (db, db_rw) = DbReaderWriter::wrap(
-        AptosDB::open(
-            &tmp_dir,
-            false,                       /* readonly */
-            NO_OP_STORAGE_PRUNER_CONFIG, /* pruner */
-            RocksdbConfigs::default(),
-            true, /* indexer */
-            TARGET_SNAPSHOT_SIZE,
+    let (db, db_rw) = if use_db_with_indexer {
+        DbReaderWriter::wrap(AptosDB::new_for_test_with_indexer(&tmp_dir))
+    } else {
+        DbReaderWriter::wrap(
+            AptosDB::open(
+                &tmp_dir,
+                false,                       /* readonly */
+                NO_OP_STORAGE_PRUNER_CONFIG, /* pruner */
+                RocksdbConfigs::default(),
+                true, /* indexer */
```

Comment:
I think this should be false here, right?

Reply:
This was already true, but I'll make it false and see if the tests pass. If so I'll leave it that way.

Reply:
Yeah indexer wants this true, other tests should have it false :-)